### PR TITLE
Introduce the DB model for the personal contribution

### PR DESCRIPTION
**Pull request by alex-voodoo**
This patch introduces the DB model for the personal contribution (related to #33).

### CODE REVIEW

File: skills/models.py [hunk 1]
```diff
@@ -154,3 +154,20 @@ class Meta:
...
+    """
+    person = models.ForeignKey(Person, on_delete=models.CASCADE)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    skills = models.ManyToManyField(Skill, blank=True)
+
+    description = models.TextField()
+
+    class Meta:
+        verbose_name = _('Contribution')
+        verbose_name_plural = _('Contributions')
```
Comment (anarute) [inline skills/models.py:170]: By default if it's a regular plural, Django [already adds the 'S' in the end of the word](https://docs.djangoproject.com/en/4.2/ref/models/options/#verbose-name-plural), so for this specific case it wouldn't be needed. But this is very minor, I don't see an issue on keeping it, just thought it was worth mentioning.

**Review by anarute — APPROVED** [review]
LGTM!

File: skills/models.py (hunk 1 above)
Comment (alex-voodoo) [reply skills/models.py:170]: Django adds 's' in the end of English (non-translated) words, but I wrap these words with a call to translator.

Comment (anarute) [reply skills/models.py:170]: Got it, it makes sense, thanks :)

**Pull request merged**